src/main/resources/aeroplan_distances.csv

```diff
@@ -647,6 +647,7 @@ SFO,YYJ,764,
 SFO,YYZ,2254,2251
 SFO,ZRH,5827,
 SIN,ZRH,6400,6404
+SIN,SYD,,3906
```

Comment:
```suggestion
SIN,SYD,,3906
SIN,ZRH,6400,6404
```

These are alphabetical

Reply:
I did a second commit instead of just committing your suggestion.  Apparently I have forgotten how to use git sometime in the last decade or so